src/main/java/org/folio/service/orders/flows/update/unopen/UnOpenCompositeOrderManager.java

```diff
@@ -142,7 +142,8 @@ private Future<Void> processInventory(CompositePoLine compPOL, boolean deleteHol
         if (compPOL.getCheckinItems()) { // independent workflow
           return (deleteHoldings) ? processInventoryOnlyWithHolding(compPOL, requestContext) : Future.succeededFuture();
         } else { // synchronized workflow
-          return processInventoryOnlyWithHolding(compPOL, requestContext);
+          return (deleteHoldings) ? processInventoryOnlyWithHolding(compPOL, requestContext) :
+            deleteExpectedPieces(compPOL, requestContext).mapEmpty();
```

Comment:
For karate tests new separate PR will be created, @yuntianhu will take care about it